cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/preview/DefaultPreviewStore.java

```diff
@@ -84,6 +88,8 @@ public class DefaultPreviewStore implements PreviewStore {
   private static final byte[] CONFIG = Bytes.toBytes("c");
   private static final byte[] APPID = Bytes.toBytes("a");
   private static final byte[] PRINCIPAL = Bytes.toBytes("p");
+  private static final Gson ENTITY_GSON = new GsonBuilder().registerTypeAdapter(EntityId.class,
```

Comment:
nit: Please check if this GSON with same registerTypeAdapter can be reused other places in this file. Some functions create GSON object each time and register the same type adapter.